tests/actions.go

```diff
@@ -424,14 +424,14 @@ func (oa *operatorActions) pdMembersReadyFn(tc *v1alpha1.TidbCluster) (bool, err
 			ns, pdSetName, *pdSet.Spec.Replicas, ns, tcName, replicas)
 		return false, nil
 	}
-	if pdSet.Status.ReadyReplicas != tc.Spec.PD.Replicas {
+	if pdSet.Status.ReadyReplicas != replicas {
```

Comment:
why? 
the `pdSet.Status.ReadyReplicas` is not contained failure pods, so we can not use `replicas` to check

Reply:
I'm not sure,